lib/git_chain/commands/branch.rb

```diff
@@ -19,7 +19,7 @@ def configure_option_parser(opts, options)
           options[:chain_name] = name
         end
 
-        opts.on("-i", "--insert", "Insert in the middle of a chain instead of starting a new one") do
+        opts.on("-i", "--insert", "Insert in middle of chain (after current branch) instead of starting a new one") do
```

Comment:
This is not necessarily true. If the user specifies a `start_point`, it will be inserted at that point. I would leave as it is.

Reply:
Nevermind. I see it no in help:  `branch <start_point> branch`. I've just never used it. I think it's fine as it is.